packages/mobile/src/screens/upload-screen/UploadingTracksScreen.tsx

```diff
@@ -0,0 +1,14 @@
+import { Screen, Text } from 'app/components/core'
+import { spacing } from 'app/styles/spacing'
+
+const messages = {
+  uploading: 'Uploading...'
+}
+
+export const UploadingTracksScreen = () => (
+  <Screen>
+    <Text style={{ alignSelf: 'center', marginTop: spacing(4) }}>
+      {messages.uploading}
```

Comment:
Should we add a loader here in place of/in addition to the text?

Reply:
long term this screen will totally be replaced with the right design, this was just to have something at least render to the screen :D